Automattic/jetpack
# My Jetpack: Show while disconnected

* Shows My Jetpack while disconnected

#### Changes proposed in this Pull Request:


* Removes check for connection from MyJetpack's initializer.
* Removes check for connection before rendering MyJetpackScreen component.
* Removes redirection when disconnected from connectionWatchter.
* Update behavior of connectivity problem preventing purchases fetch to only show if the error we get is not related to site connection to Jetpack.

#### Other information:


#### Screenshots

#### This is how My Jetpack will show when the site is not connected

<img width="897" alt="image" src="https://user-images.githubusercontent.com/746152/200888397-f6490c54-0653-4f57-8874-7f614e0ca352.png">

<img width="889" alt="image" src="https://user-images.githubusercontent.com/746152/200880002-70a6c9b1-ae9d-4ddd-9ffc-a0260637ecc1.png">

<img width="893" alt="image" src="https://user-images.githubusercontent.com/746152/200883925-a0faef6d-db3d-4367-90a6-416470242289.png">

##### This message only shows now when we can't fetch purchases due to reasons not related to the connection to WordPress.com

<img width="891" alt="image" src="https://user-images.githubusercontent.com/746152/200879709-df2724df-1d96-4fb7-bedb-33d95a236fec.png">


#### Jetpack product discussion
Incoming

#### Does this pull request change what data or activity we track or use?
Maybe

#### Testing instructions:
* Launch a [fresh site with this Jurassic Ninja link](https://jurassic.ninja/create?jetpack-beta&branches.jetpack=update/my-jetpack-show-while-disconnected&branches.jetpack-backup=update/my-jetpack-show-while-disconnected&branches.jetpack-boost=update/my-jetpack-show-while-disconnected&branches.jetpack-protect=update/my-jetpack-show-while-disconnected&branches.jetpack-search=update/my-jetpack-show-while-disconnected&branches.jetpack-social=update/my-jetpack-show-while-disconnected&branches.jetpack-starter-plugin=update/my-jetpack-show-while-disconnected&wp-debug-log&nojetpack)
* Confirm you see My Jetpack while the site is still not connected
